end-to-end-tests/install.txt

```diff
@@ -21,15 +21,18 @@ LOCAL COMMANDS:
 [INFO] Installing _/cowsay@0.1.2
 Package installed successfully to wapm_packages!
 [INFO] Package "cowsay" is uninstalled.
+[INFO] Package "cowsay" is not installed.
 [INFO] Installing mark/rust-example@0.1.9
 Global package installed successfully!
 Hello, world!
 
+[INFO] Package "mark/rust-example" is uninstalled.
 No packages found
 [INFO] Installing mark/wapm-override-test@0.1.0
 Global package installed successfully!
 Version 0.1.0
 [INFO] Installing mark/wapm-override-test@0.2.0
+Package installed successfully to wapm_packages!
 Version 0.2.0
-[INFO] Package "wapm-override-test" is uninstalled.
+[INFO] Package "mark/wapm-override-test" is uninstalled.
```

Comment:
I'm ok with this...are you ok with this?

Reply:
@MarkMcCaskey

Reply:
Yeah, I wish it had been caught before; it's weird that it broke on this branch and not the branch that changed it 🤔